css/style.css

```diff
@@ -651,9 +651,10 @@ div#tools
 {
 	display: inline;
 	float: right;
-	margin: 0;
-	margin-top: -15px;
-	margin-right: -18px;
+	margin:
+		-1.1em /* = -(1.3em [#content's padding-top] - 0.2em) */
+		-1.4em /* = -(1.6em [#content's padding-right] - 0.2em) */
+		1em 1em;
```

Comment:
Hm, <http://www.w3.org/TR/CSS21/syndata.html#tokenization> says:
> COMMENT tokens do not occur in the grammar (to keep it readable), but any number of these tokens may appear anywhere outside other tokens.

So, I think it's valid and GitHub gets it wrong.

But I could split it up, of course.